.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: node_js
 node_js:
-  - 0.10
+  - stable
+  - 0.12
```

Comment:
this can be changed to use only one version of node.js (I my computer, I'm using the latest stable version)